src/main/resources/templates/pps/onlinePaymentUnavailable.html

```diff
@@ -16,72 +16,63 @@
-                </div>
-                <p class="govuk-body"></p>
-                <h2 class="govuk-heading-m">Paying from an overseas account</h2>
                 <p class="govuk-body">
-                    You can pay through a bank transfer from an overseas account.
+                    You cannot use this online service to pay this penalty.
                 </p>
                 <p class="govuk-body">
-                    You'll need to use Companies House swift code, account number and account name.
+                    Pay by bank transfer instead, using these account details
                 </p>
-                <div class="govuk-details__text" aria-hidden="true" id="overseas-banking-details">
-                    <p class="govuk-body">Swift Code: NW BK GB 2L</p>
-                    <p class="govuk-body">Account Number (IBAN): GB 79 NWBK 607080 10031154 </p>
-                    <p class="govuk-body">Account name: Late Filing Penalties Receipts</p>
+                <div class="govuk-inset-text">Check the details carefully, as they may be different from other
+                    transactions you've made with us.
                 </div>
-                <p class="govuk-body"></p>
-                <p class="govuk-body">Email the LFP finance team at Companies House to tell them you've paid.
-                    Include in your email the:</p>
-                <ul class="govuk-list govuk-list--bullet" id="list-requirements-overseas-payments">
-                    <li>company number</li>
-                    <li>company name</li>
-                    <li>reference number on the penalty notice</li>
-                    <li>amount paid</li>
-                    <li>date paid</li>
+
+                <h2 class="govuk-heading-s">Bacs payment</h2>
+                <ul class="govuk-list govuk-list--bullet" id="banking-details">
+                    <li>
+                        <span class="govuk-!-font-weight-bold">Account name</span>:
+                        <span id="late-filing-account-name"
+                              th:text="${@environment.getProperty('penalty.bank-transfer-late-filing.account-name')}"></span>
+                    </li>
+                    <li>
+                        <span class="govuk-!-font-weight-bold">Sort code</span>:
+                        <span id="late-filing-sort-code"
+                              th:text="${@environment.getProperty('penalty.bank-transfer-late-filing.sort-code')}"></span>
+                    </li>
+                    <li>
+                        <span class="govuk-!-font-weight-bold">Account number</span>:
+                        <span id="late-filing-account-number"
+                              th:text="${@environment.getProperty('penalty.bank-transfer-late-filing.account-number')}"></span>
+                    </li>
+                    <li class="govuk-body"><b>Payment reference:</b> Use your penalty reference, so that we know which penalty you've paid.
```

Comment:
Looks like you have the colon in the wrong place and the html is a bit different for the bold style.
Should it follow how it has been done already here - https://github.com/companieshouse/penalty-payment-web/blob/master/src/main/resources/templates/pps/bankTransferLateFilingDetails.html#L67?